### PR TITLE
PrinterFS need story I add okay?

### PR DESCRIPTION
**Pull request by Xe**
ATUHORS' NOSE: completely true story.

### CODE REVIEW

**Comment by Hamcha** [comment]
IT'S ALWAYS GOOD TO HAVE PRODUCTION FEEDBACK
PLS ABIDE BY [OUR GUIDELINES](http://www.raidz3ro.com/media/catalog/product/cache/1/image/9df78eab33525d08d6e5fb8d27136e95/a/l/all-caps-geek-t-shirt-thumbnail.jpg).

**Comment by dequis** [comment]
THIS IS A SERIOUS PROJECT PLEASE BE SERIOUS

You might be looking for https://github.com/infoforcefeed/OlegDB/pull/138

**Comment by doublemarked** [comment]
قصة مذهلة. الآن فهمت كل شيء.
ملاحظة: أنا آسف ولكن، لوحة المفاتيح عالق بلغة أجنبية.

File: README.md [hunk 1]
```diff
@@ -42,3 +42,22 @@ sudo insmod PrinterFS.ko
...
+
+DAD: "SON, PRINTERFS IS SPACE. IT IS ALL OF THE SPACE AT THE CLOUD OF THE 
+SCALING VECTOR."
+
+SON: "BUT DAYD WHYFOR NOT HAVE FOOT FOR EATING!"
+
+DAD: "COMMUNIST CONUTRY IS DIFFERENT. FEAR NOT SON FOR WE WILL REJOICE OFF THE 
+YOKE OF CAPITALIST PIGDOGZ ENSLAVIN US"
+
+SON: "OH ISEE, FATHER JNERULA WILL SAVEUS FROM THE PRINTER AND GIVE US FREE AND
```
Comment (qpfiffer) [inline README.md]: s/SAVEUS/SAVE ÜS/

**Comment by kyleterry** [comment]
@doublemarked come back to IFF!

**Comment by lykkin** [comment]
:shipit:

**Comment by doublemarked** [comment]
אין לי לקוח צ'אט!וכריס הרע יכבה להקפיצי!

**Comment by lykkin** [comment]
though i think this should really reflect more realistically on @qpfiffer and his attempt to hoard pornography

**Comment by qpfiffer** [comment]
Sure whatever :package:

**Comment by kyleterry** [comment]
@doublemarked hit me up at my personal email kyle at kyleterry dot com. You can have a bouncer on my stuff. it basically runs forever.

**Comment by dequis** [comment]
:package:

**Comment by kyleterry** [comment]
:package:

**Comment by lykkin** [comment]
:shit:

**Pull request merged**